content/Education and Kits/Science Journal App/Upload-the-Science-Journal-firmware.md

```diff
@@ -59,17 +65,23 @@ Find and install the libraries by using the Library Manager (`Tools > Manage Lib
 
 1. Find and open the firmware upload sketch:
 
+   - **Nano RP2040 Connect:**
+
+      IDE 1.x: `File > Examples > INCOMPATIBLE > Arduino_ScienceKitCarrier > ScienceJournal`.
+
+      IDE 2: `File > Examples > Arduino_ScienceKitCarrier > ScienceJournal`.
+
    - **MKR WiFi 1010:**
 
-     IDE 1.x: `File > Examples > INCOMPATIBLE > Arduino_ScienceJournal > PhysicsLabFirmware`.
```

Comment:
Note that the users will also need to choose which kit the are using, not sure you mention this further up.

For IDE 2: 

File > Examples > Arduino_ScienceJournal > ScienceKit > PhysicsLabFirmware
![image](https://github.com/arduino/help-center-content/assets/66409231/c2507c94-738f-482e-b6e4-4adbb8430af2)

File > Examples > Arduino_ScienceJournal > ScienceKitR2 > PhysicsLabFirmware
![image](https://github.com/arduino/help-center-content/assets/66409231/cad26055-3809-479f-acd8-3041e24823ff)